sda/internal/helper/helper.go

```diff
@@ -153,6 +154,50 @@ func CreateRSAkeys(prPath, pubPath string) error {
 	return nil
 }
 
+func CreateSSHKey(path string) error {
```

Comment:
It seems this function is only called by storage_test.go, so better move it there? Especially since x509.EncryptPEMBlock is deprecated and therefore we wouldn't want to actually use this function anywhere else.

Reply:
Since the `helper.go` seems primarily used by the tests it's better to move the one function that's used in `proxy.go` to that file instead.